packages/caver-klay/caver-klay-accounts/src/makeRawTransaction.js

```diff
@@ -144,7 +144,7 @@ function makeRawTransaction(rlpEncoded, sig, transaction) {
     }
     case 'LEGACY':
     default:
-      rawTx = decodedValues.slice(0, 6).concat(sig)
+      rawTx = decodedValues.slice(0, 6).concat(sig[0])
```

Comment:
Just wondering, you returned first signatures when it is legacy transaction
at line 298 
https://github.com/klaytn/caver-js/pull/107/files#diff-e6b221946c9583fd0a407bd90c74ef39R298

In that case, you need to use sig, not sig[0].

Reply:
This makeRawTransaction function is called in signTransaction when make rawTrasnaction.
In signTransaction, just send signatures array to this function.
So in makeRawTransaction function, if transaction is legacy, just use 0th signature of signatures.